starport/pkg/localfs/search_test.go

```diff
@@ -0,0 +1,115 @@
+package localfs
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func setupGlobTests(t *testing.T, files []string) string {
+	t.Helper()
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	tmpdir, err := os.MkdirTemp(dir, "glob-test")
```

Comment:
```suggestion
	tmpdir, err := os.MkdirTemp("", "glob-test")
```

We can directly use OS' temp dir (/tmp).